imagersite/imager_profile/templates/imager_profile/profile.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+
+{% block content %}
+{% if user.username == profile.user.username %}
+<a href="{% url "library" %}">{{user.username}}'s Library</a>
+{% endif %}
+<p>{{user.username}}'s album count: {{profile.user.albums.count}}</p>
```

Comment:
This is the username of the currently logged in user. Thus it will display the wrong username if a user goes to some other user's profile.

Reply:
done